apps/swap-service/src/lib/tx-lookup.service.ts

```
@@ -0,0 +1,129 @@
+
+    // no rpc node urls exist for these, so unchained is the only route
+    const utxoNodes: [ChainId, string][] = [
+      [caip.btcChainId, env.VITE_UNCHAINED_BITCOIN_HTTP_URL],
+      [caip.bchChainId, env.VITE_UNCHAINED_BITCOINCASH_HTTP_URL],
+      [caip.dogeChainId, env.VITE_UNCHAINED_DOGECOIN_HTTP_URL],
+      [caip.ltcChainId, env.VITE_UNCHAINED_LITECOIN_HTTP_URL],
+    ]
+
+    for (const [chainId, basePath] of utxoNodes) {
+      const api = new unchained.bitcoin.V1Api(new unchained.bitcoin.Configuration({ basePath }))
+      this.fetchers.set(chainId, unchainedFetcher((req) => api.getTransaction(req)))
```

ESLint

[error] Replace `chainId,·unchainedFetcher((req)·=>·api.getTransaction(req))` with `⏎········chainId,⏎········unchainedFetcher((req)·=>·api.getTransaction(req)),⏎······`

(prettier/prettier)